tests/trainer/test_trainer.py

```diff
@@ -62,6 +62,10 @@
 if _OMEGACONF_AVAILABLE:
     from omegaconf import OmegaConf
 
+ProcessRaisedException = Exception
+if _TORCH_GREATER_EQUAL_1_8:
+    from torch.multiprocessing import ProcessRaisedException
```

Comment:
```suggestion
if _TORCH_GREATER_EQUAL_1_8:
    from torch.multiprocessing import ProcessRaisedException
else:
    ProcessRaisedException = Exception
```

just a nit, but doesn't make sense to have a value and immediately overwrite it :)